packages/preview/athena-tu-darmstadt-thesis/0.1.0/templates/tudapub/tudapub.typ

```diff
@@ -0,0 +1,663 @@
+#import "tudacolors.typ": tuda_colors
+#import "common/outline.typ": *
+#import "common/props.typ": *
+#import "common/tudapub_title_page.typ": *
+#import "common/thesis_statement_pursuant.typ": *
+#import "util.typ": *
+
+#import "@preview/i-figured:0.2.3"
+
+
+
+// This function gets your whole document as its `body` and formats it.
+#let tudapub(
+  title: [Title],
+  title_german: [Title German],
```

Comment:
Yes. I am planning to change this in a future version. 
But for the first version, I want to keep the names as-is, since I and at least one other person use the package locally with the current snake_case. I want to ensure that the local versions can be replaced with the universe version without breaking anything.